cloudinary/apiCaller.js

```diff
@@ -11,25 +11,31 @@ const _ = require('lodash');
 const cloudinaryParser = require('./cloudinaryResultParser');
 const cloudinary = require('cloudinary');
 const async = require('async');
-const request = require('request');
+const request = require('got');
```

Comment:
Do we want to change this across the board? kinda weird having two request libraries no?